Documentation/contributing/development/dev_setup.rst

```diff
@@ -9,6 +9,19 @@
 Development Setup
 =================
 
+Dev Container
+~~~~~~~~~~~~~
+
+Cilium provides Dev Container configuration for  Visual Studio Code Remote Containers
+and Github Codespaces. This allows you to use a preconfigured development environment
+in the cloud or locally. The container is based on the official Cilium builder image
+and provides all the dependencies required to build Cilium.
+
+.. note::
+
+    The Dev Container configuration might not have some limitations, such as running
+    integration tests locally. Please refer to the next section for more information.
```

Comment:
Does it have limitations or not? This says "might not have some limitations", which seems like a weird note to leave for readers. I'm also not sure which part of the next section is supposed to provide more information :thinking: